src/meta/languages.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
```

Comment:
Can you add comments to this file? Things like

`// Generate fallback language strings for plugin defined language files`?